_assets/ci/Jenkinsfile.tests

```
@@ -115,6 +115,11 @@ pipeline {
         reportName:             'Reports',
         reportTitles:           'Test Stats'
       ]
+      script {
+        if (isTestNightlyJob()) {
+          archiveArtifacts artifacts: '**/report.xml, **/test.log'
```

You could just do:
```suggestion
          archiveArtifacts(artifacts: '**/report.xml, **/test.log', allowEmptyArchive: true)
```
And avoid the conditional entirely.
Also, the whole `always` block might as well start with a `script` block.

---

I don't think we need to publish xml and log files for PR runs.
They use space, that's why we decided to hide them from STDOUT.

---

I see, fair enough. How big are they tho?

---

I think they will be twice big as they were in stdout before:
https://github.com/status-im/status-go/issues/3543#issuecomment-1717264766
So I estimate 100M per PR build.

For nightly it will be 100M * 20 = 2G

---

Then the conditional makes sense. Thanks.